<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nika</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 59, 89);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ListenBTN">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad, x1:0.278409, y1:0.739, x2:0.863636, y2:0.153, stop:0 rgba(131, 136, 164, 255), stop:1 rgba(137, 91, 237, 255));
border: 2px solid rgb(210, 234, 255);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>files/micro.png</normaloff>files/micro.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="outLabel">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>190</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: None;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="MenuBTN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;toolButtonStyle: Qt::ToolButtonIconOnly;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #8388a4; /* Цвет фона */
border: 1px solid #8388a4; /* Убираем границу кнопки */
border-radius: 25px;
padding: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>files/menu.png</normaloff>files/menu.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
